cime_config/SystemTests/pvt.py

```diff
@@ -0,0 +1,131 @@
+"""
+FATES land use potential vegetation spin up + transient test
+
+This is a FATES specific test:
+
+1) conduct a spinup with use_fates_potentialveg on
+    - write restart file
+    - use CLM_ACCELERATED_SPINUP?
```

Comment:
While AD will likely be used for spin-up the main goal of the test is to exercise this `use_fates_potentialveg` to transient fates `use_fates_luh` mode.  Per discussion with @ekluzek it would be good to include a test that exercises going from AD to `use_fates_potentialveg`.

Reply:
See #2648